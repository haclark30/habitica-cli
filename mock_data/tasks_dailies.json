{
    "data": [
        {
            "type": "daily",
            "text": "test daily 1",
            "completed": true,
            "streak": 5
        },
        {
            "type":"daily",
            "text": "test daily 2",
            "completed": false,
            "streak": 2
        }
    ]
}