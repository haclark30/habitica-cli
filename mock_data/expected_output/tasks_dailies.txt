[x] test daily 1 (5 days in a row)
[ ] test daily 2 (2 days in a row)
